doc/install.txt

```diff
@@ -32,30 +28,19 @@ Setup for GNU/Linux
 On all major Linux distributions (Debian, Mandriva, Redhat, Suse, Ubuntu),
 the ``linkchecker`` package is available for install.
 
-
 Manual setup for Unix systems
 -----------------------------
 First, install the required software.
 
-1. You need a standard GNU development environment with
-   
-   - C compiler (for example the GNU C Compiler gcc)
-     
-     Depending on your distribution, several development packages
-     might be needed to provide a fully functional C development
-     environment.
-   
-   Note for developers: if you want to regenerate the po/linkchecker.pot
-   template from the source files, you will need xgettext with Python
-   support. This is available in gettext >= 0.12.
-
-2. Python >= 2.7.2 from http://www.python.org/
+1. Python >= 3.5.0 from http://www.python.org/
    
    Be sure to also have installed the included distutils module.
    On most distributions, the distutils module is included in
    an extra ``python-dev`` package.
 
-3. Python requests module from https://pypi.python.org/pypi/requests
+2. Python requests package from https://pypi.python.org/pypi/requests
```

Comment:
PyPI is now using https://pypyi.org/project/... URLs, but I wonder if we want to encourage manual installation of dependencies?

Reply:
Updated the URL. Manual is the third option. Where else do we document the optionals? Indeed pip isn't going to help with those

Reply:
I can't help noticing "the URL", singular.  There were multiple pypi.python.org URLs in the text; I didn't highlight them all.

Reply:
I have updated them, not posted yet, and converted GeoIP and mod_wsgi to PyPI. And more htttps.